src/Components/Clock.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  today: Date;
+};
+
+export class Clock extends Component< Props, State > {
+  state = {
+    today: new Date(),
+  };
+
+  dateTimer = 0;
+
+  componentDidMount() {
+    this.dateTimer = window.setInterval(() => {
+      this.setState({
+        today: new Date(),
+      });
+
+      window.console.info(this.state.today.toUTCString().slice(-12, -4));
+    }, 1000);
```

Comment:
You don't need `window`
```suggestion
      console.info(this.state.today.toUTCString().slice(-12, -4));
    }, 1000);
```